ui/app/AppLayouts/Communities/popups/FinaliseOwnershipPopup.qml

```diff
@@ -0,0 +1,335 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQml.Models 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Popups.Dialog 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core.Utils 0.1
+
+import AppLayouts.Communities.panels 1.0
+
+import utils 1.0
+
+StatusDialog {
+    id: root
+
+    // Community related props:
+    required property string communityName
+    required property string communityLogo
+    required property string communityColor
+
+    // Owner token related props:
+    required property string tokenSymbol
+    required property string tokenChainName
+
+    // Fees calculation props:
+    property string feeText
+    property string feeErrorText
+    property bool isFeeLoading
+    property string feeLabel: qsTr("Update %1 Community smart contract on %2").arg(root.communityName).arg(root.tokenChainName)
+
+    // Account expected roles: address, name, color, emoji, walletType
+    property var accounts
+
+    signal finaliseOwnershipClicked
+    signal rejectClicked
+    signal visitCommunityClicked
+    signal openControlNodeDocClicked(string link)
+
+    QtObject {
+        id: d
+
+        readonly property string controlNodeLink: "https://help.status.im/en/status-communities/about-the-control-node-in-status-communities/"
```

Comment:
Are you sure the help URL is correct? We already have a different one in `Constants.statusHelpLinkPrefix`

Reply:
Yeah! Updated to: `https://status.app/help/status-communities/about-the-control-node-in-status-communities`